### ISSUE TITLE
VkDrawIndexed validation layer suggestion

### ISSUE THREAD

**Issue by UPS7kYkHEPbZIucaqjje** [KhronosGroup/Vulkan-LoaderAndValidationLayers#2086]
Basically:
Created only created pipeline once (which needs a renderpass) because it is a dynamic pipeline

later on

On window resizing 
    destroyed renderpass and recreated it
    recreated commandbuffers
    VkDrawIndexed throws an exception that has no validation layer attached or message. The exception only happens on debug mode. Because the pipeline is referencing an old renderpass.

Can a validation layer be added to check this behavior? Or do I need to add more information?

**Comment by tobine** [comment]
@UPS7kYkHEPbZIucaqjje If I understand your complaint correctly it sounds like a driver bug. Here's [spec](https://www.khronos.org/registry/vulkan/specs/1.0-extensions/html/vkspec.html#fundamentals-objectmodel-overview) text on renderPass lifetime:

> A VkRenderPass object passed as a parameter to create another object is not further accessed by that object after the duration of the command it is passed into.

**Comment by UPS7kYkHEPbZIucaqjje** [comment]
Let me try and explain again.

I created an initial dynamic pipeline with a renderpass (lets call it renderpass_1) I created beforehand.

Then on window resizing, I destroyed and recreated the renderpass_1 with the same parameters. Now I have a renderpass_2. 

Now when I recreate the commandbuffers the VkDrawIndexed throws an exception. Presumably because when attaching the pipeline again the pipeline has a reference to renderpass_1. While the only valid renderpass is renderpass_2.
